lib/index.js

```diff
@@ -103,6 +94,9 @@ Outbound.prototype.identify = function(identify) {
  */
 
 Outbound.prototype.track = function(track) {
+  if (!this.hasIdentified()) {
+    this.identify(track.userId() || track.anonymousId());
```

Comment:
@sperand-io will the `identify` facade object be available to me in a `track` call?